spec/components/organisation_logo_spec.rb

```diff
@@ -71,4 +71,9 @@ def component_name
     render_component(organisation: { name: "Name", url: "/some-link" }, inline: true)
     assert_select "a.gem-c-organisation-logo__container--inline"
   end
+
+  it "renders on a dark background" do
+    render_component(organisation: { name: "Name", url: "/some-link" }, inverse: true)
+    assert_select ".gem-c-organisation-logo.gem-c-organisation-logo--inverse"
+  end
```

Comment:
Nitpick: I also like to add another test that checks that the class doesn't exist when `inverse` is `false`, just so we're ensured classes are definitely tied to their boolean being `true`, and I think technically is better code coverage. (I appreciate the other examples don't have this type of test, so feel free to ignore this)

Reply:
No problem, can add that in